app/authentication/admin.py

```diff
@@ -0,0 +1,5 @@
+# Register your models here.
+from .models import User
```

Comment:
This is not how you import User model as discussed

Reply:
Done. Check it out